packages/flutter/lib/src/services/autofill.dart

```diff
@@ -693,15 +692,16 @@ class AutofillConfiguration {
   final List<String> autofillHints;
 
   /// The current [TextEditingValue] of the [AutofillClient].
-  final TextEditingValue currentEditingValue;
+  final TextEditingValue? currentEditingValue;
```

Comment:
/cc @LongCatIsLooong When is this expected to be null? Can we add that to the documentation here?

Reply:
It's not supposed to be null but I might have omitted `currentEditingValue` in some of the tests.

Reply:
By making `currentEditingValue` required and non-nullable tests on packages/flutter are OK.

I made it nullable because in the constructor it was not `@required` and not assered as not null.

I'll make the change to have non-nullable type.